amundsen_application/static/js/components/TableDetail/TableIssues/index.tsx

```diff
@@ -58,68 +58,67 @@ export class TableIssues extends React.Component<TableIssueProps> {
     );
   }
 
-  renderMoreIssuesMessage = (count: number, url: string) => {
+  renderIssueTitle = () => {
     return (
-      <span className="table-more-issues" key="more-issue-link">
-        <a id="more-issues-link" className="table-issue-more-issues" target="_blank" href={url} onClick={logClick}>
-         View all {count} issues
-        </a>
-        |
-        { this.renderReportIssueLink() }
-      </span>
+      <div className="section-title title-3">
+        Issues
+      </div>
     );
   }
 
-  renderReportIssueLink = () => {
-    return (
-      <div className="table-report-new-issue">
+  renderIssueContent = () => {
+    if (this.props.issues.length === 0) {
+      return (
+        <div className="issue-banner">
+          { NO_DATA_ISSUES_TEXT }
+        </div>
+      )
+    };
+    return this.props.issues.map(this.renderIssue);
+  }
+
+  renderIssueFooter = () => {
+    const hasIssues = this.props.issues.length !== 0;
+    const reportIssueLink = (
+      <div className={`table-report-new-issue ${hasIssues ? 'ml-1' : ''}`}>
         <ReportTableIssue tableKey={ this.props.tableKey } tableName={ this.props.tableName }/>
       </div>
     );
```

Comment:
Nit: Maybe we can get rid of the "reportIssueLink" temporal value and return this markup when there are no issues?

Reply:
I return this on either line 89 or line 97, so created this value so that we only have one place to touch code when updating this. Are there any downsides to the temporal value to consider? (whether it be in this PR or just generally when it comes to writing code in React)

Reply:
Oh, sorry, I didn't see the one on line 97.

Not many downsides in reality, just that it implies that it is being reused (and I didn't see that before)

Thanks!